README.md

```diff
@@ -181,8 +181,11 @@ try {
 }
 ```
 
-This mechanism could even be codified into `import()` by making it reject if
-the target module has not run to end of source text in Evaluation yet. At the current time [a search](https://github.com/search?utf8=%E2%9C%93&q=%22export+async+function%22&type=Code) for "export async function" on github produces over 5000 unique code examples of exporting an async function.
+While it is tempting to codify this behavior by specifying that `import()` should reject if the imported module has not finished evaluating, that behavior would make `import()` a poor abstraction for importing asynchronous modules, since it would deprive well-behaved modules of the chance to finish async work, just because they were imported using `import()`.
```

Comment:
disagree. choice of poor/not seems to be a distracting statement. the concept well-behaved also seems to be chosen without definition.

Reply:
Then I can use different words and/or elaborate on the implied definitions of the words I used. I wish that your feedback here was more collaborative.

Reply:
I really don't agree with any of the statements, I'm not sure how I could be rephrasing it. I don't think it has any good/bad connotations without further examination.